src/modules/missionHelper/i18n/cs_CZ.root.json

```diff
@@ -15,6 +15,10 @@
         "title": "Special requirements"
     },
     "settings": {
+        "hide_on_Krankentransport": {
+            "description": "Hide on ambulance missions",
+            "title": "Hide on ambulance missions"
```

Comment:
I would suggest/prefer "patient transfer missions". (Same for other EN translations)